ports/esp32/boards/LOLIN_S2_MINI/modules/s2mini.py

```diff
@@ -0,0 +1,52 @@
+# LOLIN S2 MINI MicroPython Helper Library
+
+
+from micropython import const
+from machine import Pin, SPI, ADC
+import machine, time
+
+import network
+
+# Pin Assignments
+
+
+# SPI
+SPI_MOSI = const(11)
+SPI_MISO = const(9)
+SPI_CLK = const(7)
+
+# I2C
+I2C_SDA = const(33)
+I2C_SCL = const(35)
+
+# DAC
+DAC1 = const(17)
+DAC2 = const(18)
+
+# LED
+LED = const(15)
+
+# BUTTON
+BUTTON = const(0)
+
+
+class led:
```

Comment:
Is there a reason to make this class name lowercase?  Usually classes are upper camel case, which in this situation would be `LED`.

Also, this is a very simple class and `machine.Pin` already provides the on/off methods.  In my opinion it would be simpler to pre-create the LED object like this:
```python
led = Pin(LED, Pin.OUT, value=0)
```
Then the user can just do `s2mini.led.on()`.